acstools/tests/test_wfc_findsat_mrt.py

```diff
@@ -0,0 +1,39 @@
+'''
+Test satellite trail detection and masking using findsat_mrt
+'''
+
+from acstools.findsat_mrt import wfc_wrapper
+from acstools.tests.helpers import BaseACSTOOLS
+import acstools as a
+package_path = a.__path__[0]
+import os
+
+#update all this with my code
+
+class test_findsat_mrt(BaseACSTOOLS):
+    detector = 'wfc'
+    package_directory = os.path.dirname(os.path.abspath(__file__))
+
+    def test_wfc_wrapper(self):
+        """Identify and mask trails in WFC extension 4."""
+
+        rootname = 'jc8m32j5q' 
+        inputfile = rootname + '_flc.fits'  
+        truthmaskfile = rootname + '_flc_mask_ref.fits'
+        truthcatalogfile = rootname + '_flc_catalog_ref.fits'
+
+        # Prepare input file.
+        self.get_input_file(inputfile, skip_ref=True)
+
+        wfc_wrapper(inputfile, binsize=4, extension=4,
+                    output_root='jc8m32j5q',
+                    output_dir = package_path + '/tests/data/input/',
```

Comment:
Use tmp_path if possible: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Prefer to leave as is for the following reason: I'm trying to use the "compare_outputs" code which appears to only look in the /data/input and /data/truth/ directories for the files to test. So it is easier if my code writes the output files to the /data/input/ directory.

Reply:
Do you really need `compare_outputs` though. I haven't looked at your code in depth but that test class was written for HSTCAL that only can write to "working directory".

Anyways, if you are adamant on this, you have to make sure Git ignore those output files so you don't accidentally check them into version control.

Reply:
I modified the test to not use compare_outputs and use the temporary directory for the output files.